app/router.js

```diff
@@ -9,6 +9,7 @@ const Router = Ember.Router.extend({
 Router.map(function() {
   this.route('reminders', function(){
     this.route('reminder', { path: '/:id' });
+    this.route('new', { path: '/new'});
```

Comment:
because the path is exactly the same as the route name you can remove the `{ path :... }` bit. In other words, this line can simply be `this.route('new')`